frontend/scss/molecules/_m-custom-banner.scss

```diff
@@ -151,117 +151,22 @@
             background-color: #ffffff20 !important;
             }
         }
     }
 
-    &.custom-banner-block--editorial {
-        margin-top: 50px;
-        padding-top: 50px;
-        border-top: 1px solid $color__black--20;
-
-        .background-wrapper {
-            &::before {
-                background: $color__cold-white;
-            }
-        }
-
-        &.custom-banner-block--variation-cloud {
-            .content-wrapper {
-                height: 276px;
-                display: flex;
-            }
-
-            .text-wrapper {
-                align-items: center;
-                height: unset;
-                
-                @include breakpoint('medium-') {
-                    align-items: flex-start;
-                }
-
-                >div:first-child {
-                    align-self: baseline;
-                }
-            }
-
-            .button-wrapper {
-                max-width: 70%;
-            }
-
-            .custom-banner-title {
-                font-family: $sans-serif-font--loaded;
-                font-size: 21px;
-                font-weight: 400;
-                line-height: 28px;
-                letter-spacing: 0.08em;
-                text-align: left;
-                text-transform: uppercase;
-                color: $color__black--90 !important;
-            }
-
-            .custom-banner-body {
-                font-family: $serif-font--loaded;
-                font-size: 22px;
-                font-weight: 300;
-                line-height: 28px;
-                letter-spacing: -0.005em;
-                text-align: left;
-                color: $color__black--90 !important;
-            }
-
-            .tag-cloud {
-                display: flex;
-                justify-content: center;
-                flex-wrap: wrap;
-                align-items: center;
-
-                @include breakpoint('medium-') {
-                    display: none;
-                }
-
-                .f-tag {
-                    background-color: $color__white;
-                    padding: 6px 12px;
-                    margin: 6px;
-                    font-family: $sans-serif-font--loaded;
-                    font-size: 12px;
-                    font-weight: 400;
-                    line-height: 16px;
-                    letter-spacing: 0.14em;
-                    color: $color__black;
-                    border: 1px solid $color__black--20;
-                    border-radius: 2px;
-                }
-            }
-
-            .tag-cloud__link {
-                display: block;
-                margin: auto;
-                color: $color__rules--brand;
-                text-align: center;
-                font-size: 20px;
-                font-weight: 300;
-                line-height: 28px;
-                letter-spacing: 0.0001em;
-                text-transform: none;
-                padding-top: 17px;
-                white-space: nowrap;
-            }
-        }
-    }
-}
```

Comment:
All of this became the default style of the tag banner, with some modifications.